sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1709,6 +1718,10 @@ object GpuOverrides extends Logging {
               }
             case _ => // NOOP
           }
+
+          if (SQLConf.get.ansiEnabled) {
+            willNotWorkOnGpu("GPU Multiplication does not support ansi mode")
```

Comment:
```suggestion
            willNotWorkOnGpu("GPU Multiplication does not support ANSI mode")
```

Reply:
I'll do this in a follow on issue as @firestarman is now out on vacation.